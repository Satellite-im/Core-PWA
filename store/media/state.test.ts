import { expect } from '@jest/globals'
import InitialMediaState from '~/store/media/state'

describe('init', () => {
  let inst: any

  beforeEach(() => {
    inst = InitialMediaState()
  })

  it('should return the initial media state', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the initial media state', () => {
    expect(inst).not.toEqual({})
  })
})
