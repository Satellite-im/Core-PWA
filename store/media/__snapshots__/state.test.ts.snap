// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should return the initial media state 1`] = `
Object {
  "activeCall": "0x0",
  "incomingCall": "",
  "streaming": false,
}
`;
